tools/tasks/tslint.ts

```diff
@@ -11,7 +11,9 @@ export = function tslint(gulp, plugins) {
     ];
 
     return gulp.src(src)
-      .pipe(plugins.tslint())
+      .pipe(plugins.tslint({
+        rulesDirectory: NG2LINT_RULES
```

Comment:
Yep. For VSCode it should actually be added to `.vscode/settings.json` (https://marketplace.visualstudio.com/items?itemName=eg2.tslint)